### DISCUSSION TITLE
How to add a nginx to proxy_pass

### DISCUSSION THREAD

**Discussion by Chunshan-Theta** [infiniflow/ragflow#2087] [Q&A]
I want to build a nginx as proxy, this is my nginx config:
```
events {}

http {
    server {
        listen 80;

        location /ragflow/ {
            proxy_pass http://127.0.0.1:8001;
            proxy_set_header Host $host;
            proxy_set_header X-Real-IP $remote_addr;
            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
            proxy_set_header X-Forwarded-Proto $scheme;
            #rewrite ^/ragflow(/.*)$ $1 break;  # Strips /ragflow/ from the proxied request path
        }
    }
}

```
but I try http://{domainName}/ragflow/ , then return `GET http://{domainName}/umi.0e8dd54d.js net::ERR_ABORTED 404 (Not Found)` in the browser dev tool console.

p.s.  I had already deployed ragflow on 8001

**Comment by cike8899** [comment]
You can do like this:
```
server {
    listen       80;
    server_name {domainName};
    location / {
        proxy_pass http://localhost:8001;
    }
}
```
Access via `http://{domainName}`

**Reply by Chunshan-Theta** [reply]
possiable to use  http://{domainName}/ragflow/? because we have other services on  the same site

**Reply by cike8899** [reply]
You can use nginx to configure different subdomains for different sites.

**Comment by shuaishuaishuaishuaishuaishuai** [comment]
同问，请问作者解决了吗
